### PR TITLE
fixes order_by_u64_field and order_by_fast_field should allow sorting in ascending order #1676

### PR DESCRIPTION
**Pull request by naveenann**

### CODE REVIEW

File: src/collector/top_score_collector.rs [hunk 1]
```diff
@@ -1106,4 +1129,44 @@ mod tests {
...
+                ))
+                .unwrap();
+            index_writer
+                .add_document(doc!(
+                    title => "growler of beer",
+                    size => 64u64,
+                ))
+                .unwrap();
+            index_writer
+                .add_document(doc!(
```
Comment (PSeitz) [inline src/collector/top_score_collector.rs:1152]: can you add an empty doc to the test
```rust
index_writer.add_document(doc!()).unwrap();
```

Comment (PSeitz) [reply src/collector/top_score_collector.rs:1152]: sorry I meant a doc without a value

```rust
index_writer.add_document(doc!(title => "empty beer"))
```

Comment (naveenann) [reply src/collector/top_score_collector.rs:1152]: ahh ok. without the size field.

File: src/collector/top_score_collector.rs [hunk 2]
```diff
@@ -157,8 +171,13 @@ impl CustomScorer<u64> for ScorerByField {
             sort_column_opt.ok_or_else(|| FastFieldNotAvailableError {
                 field_name: self.field.clone(),
             })?;
+        let mut value = 0u64;
```
Comment (PSeitz) [inline src/collector/top_score_collector.rs]:
```suggestion
        let mut default_value = 0u64;
```

**Review by PSeitz — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**